workflows/mtbseqnf.nf

```diff
@@ -44,16 +62,66 @@ workflow MTBSEQ_NF {
                                 params.intregions,
                                 params.categories,
                                 params.basecalib])
-           //     ch_versions = ch_versions.mix(NORMAL_ANALYSIS.out.versions)
 
+                ch_versions =  ch_versions.mix(NORMAL_ANALYSIS.out.versions)
+                ch_multiqc_files =  ch_multiqc_files.mix(NORMAL_ANALYSIS.out.multiqc_files)
+
+        }
     }
-    /// END MTBSEQ ANALYSIS
 
-    REPORT (QC.out.ch_multiqc_files, ch_versions)
-    multiqc_report = REPORT.out.multiqc_report
+    //
+    // Collate and save software versions
+    //
+    softwareVersionsToYAML(ch_versions)
+        .collectFile(
+            storeDir: "${params.outdir}/pipeline_info",
+            name: 'nf_core_pipeline_software_mqc_versions.yml',
+            sort: true,
+            newLine: true
+        ).set { ch_collated_versions }
+
+    //
+    // MODULE: MULTIQC
+    //
+    ch_multiqc_config        = Channel.fromPath(
+        "$projectDir/assets/multiqc_config.yml", checkIfExists: true)
+    ch_multiqc_custom_config = params.multiqc_config ?
```

Comment:
Had to move `MULTIQC` to top-level as an encapsulated `REPORT` workflow was introducing it's own complexity and once the baseline module was used the issues with channel content and plugin function imports were resolved.